app/src/utils/hooks.ts

```diff
@@ -220,6 +221,31 @@ export const useTestingEntries = (refetchInterval?: number) => {
   return useStableData(result);
 };
 
+export const useStats = (
```

Comment:
I'm a little confused about the behavior here. It looks like if a user sets custom dates, we define the range we show as the first day of the month of the start date to the last day of the month of the end date? That seems like surprising/unexpected behavior to me -- why not just define the range as the beginning of the day of the start date to the end of the day at the end date?

Reply:
Yes, it was my typo.
Changing:
startOf("month") -> startOf("day")
endOf("month") -> endOf("day")